backend/src/routes/player.route.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from "express";
+import { createPlayer } from "../controllers/player.controller";
+
+const router = Router();
+
+router.get("/ping", async (req, res) => res.sendStatus(200));
```

Comment:
Were the ping routes just for testing?

Reply:
Mostly yep - still useful but yeah mainly testing. Should I get rid of them?

Reply:
Yeah I think we should remove them if they're not being used 👌🏼